### PR TITLE
fix for create assistant phone number input field and business card

### PR DESCRIPTION
**Pull request by TokyoTG**
# Description

Please include a summary of the change and which issue is fixed. Add a refrence to the issue that was fixed! 

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnin

### CODE REVIEW

File: config/snappy.php [hunk 1]
```diff
@@ -49,8 +49,8 @@
     
     'image' => [
         'enabled' => true,
-        'binary'  => env('WKHTML_IMG_BINARY', '/usr/local/bin/wkhtmltoimage'),
-        // 'binary' => '"C:\Program Files\wkhtmltopdf\bin\wkhtmltoimage"',
+        // 'binary'  => env('WKHTML_IMG_BINARY', '/usr/local/bin/wkhtmltoimage'),
+        'binary' => '"C:\Program Files\wkhtmltopdf\bin\wkhtmltoimage"',
```
Comment (Douglasokolaa) [inline config/snappy.php:53]: can you verify this is okay

Comment (kofimokome) [reply config/snappy.php:53]: @TokyoTG I told you to change this to accept from the environment file

Comment (TokyoTG) [reply config/snappy.php:53]: not okay

Comment (TokyoTG) [reply config/snappy.php]: @Douglasokolaa  have made the changes

**Comment by kofimokome** [comment]
Can you make the slideshow to pause when I hover over the business card.
I clicked on the other card because the slide changed before I clicked

**Comment by TokyoTG** [comment]
k i will check bootstrap

**Pull request merged**